src/context/ChartContextProvider.tsx

```diff
@@ -155,16 +153,64 @@ function useTidyDataToChartContext(
       };
 
       // The Y values object also passes through the user specified series properies from dataSelection
-      const yValues = {
+      let yValues = {
         name: series.displayName,
         color: series.color,
         dashStyle: series.dashStyle,
+        intervalType: "---",
```

Comment:
Do we use the '---' option in other chart builder config? If it's keeping us consistent with a standard then fine, but I wonder if 'None' would actually be more descriptive?

Reply:
dashStyle uses 'None' but thats not visible, Data columns uses "--", so I've changed it to that.